env/setenv.sh

```diff
@@ -74,8 +74,14 @@ function choose_khadas_board() {
 	i=0
 
 	KHADAS_BOARD_ARRAY=()
-	for board in $ROOT/config/boards/*.conf; do
-		KHADAS_BOARD_ARRAY+=("$(basename $board | cut -d'.' -f1)")
+
+#	for board in $ROOT/config/boards/*.conf; do
+#		KHADAS_BOARD_ARRAY+=("$(basename $board | cut -d'.' -f1)")
+#	done
+
+#	1 VIM1 2 VIM2 ..... 
+	for board in VIM1 VIM2 VIM3 VIM3L Edge; do
```

Comment:
Why do this ? If new board is added you have to edit this file again. So I think the original way is better, auto scan the board files.

Reply:
may be ! its just my imho!

Reply:
every time i press 1 by intuitive automatically - think its will be vim1
or press 2 and wait it will be vim2 but its vim1 
!!!
u can reject this one - its just my vision! not really important

Reply:
> every time i press 1 by intuitive automatically - think its will be vim1
> or press 2 and wait it will be vim2 but its vim1

There maybe have Edge 2, Edge 3 .... in the future, so you can't bind 1 to VIM1